packages/pipeline-editor/src/PipelineExportDialog.tsx

```diff
@@ -60,7 +60,12 @@ export class PipelineExportDialog extends React.Component<IProps> {
             </option>
           ))}
         </select>
-        <input type="checkbox" id="overwrite" name="overwrite" />
+        <input
+          type="checkbox"
+          id="overwrite"
+          name="overwrite"
+          style={{ appearance: 'auto', height: 'auto' }}
+        />
```

Comment:
Is there a way we can make this global in a CSS? otherwise, you will need to fix it in `submit notebook` and possibly on the node property dialog box.

Reply:
Yes, good point, I thought this was the only checkbox affected. The one in node properties look good, but you're right, we have the same issue in `submit notebook`.